### ISSUE TITLE
Precompiling TypedTables hangs with Julia v1.2.0-DEV.669

### ISSUE THREAD

**Issue by oschulz** [FugroRoames/TypedTables.jl#48]
Not sure if it's a Julia issue, or something to do with TypedTables: With Julia v1.2.0-DEV.669 (2019-04-09), precompilation of TypedTables` hangs. Using an empty `~/.julia` directory:

```
julia> versioninfo()
Julia Version 1.2.0-DEV.669
Commit 4671132ee1 (2019-04-09 19:17 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)

julia> using Pkg

julia> pkg"add TypedTables"
   Cloning default registries into `~/.julia`
   Cloning registry from "https://github.com/JuliaRegistries/General.git"
     Added registry `General` to `~/.julia/registries/General`
 Resolving package versions...
 Installed IteratorInterfaceExtensions ─ v0.1.1
 Installed TableTraits ───────────────── v0.4.1
 Installed Requires ──────────────────── v0.5.2
 Installed Indexing ──────────────────── v1.1.0
 Installed Tables ────────────────────── v0.1.18
 Installed TypedTables ───────────────── v1.1.0
 Installed SplitApplyCombine ─────────── v0.4.1
  Updating `/tmp/oschulz/juliadir/environments/v1.2/Project.toml`
  [9d95f2ec] + TypedTables v1.1.0
  Updating `/tmp/oschulz/juliadir/environments/v1.2/Manifest.toml`
  [313cdc1a] + Indexing v1.1.0
  [82899510] + IteratorInterfaceExtensions v0.1.1
  [ae029012] + Requires v0.5.2
  [03a91e81] + SplitApplyCombine v0.4.1
  [3783bdb8] + TableTraits v0.4.1
  [bd369af6] + Tables v0.1.18
  [9d95f2ec] + TypedTables v1.1.0
  [2a0f44e3] + Base64 
  [8ba89e20] + Distributed 
  [b77e0a4c] + InteractiveUtils 
  [8f399da3] + Libdl 
  [37e2e46d] + LinearAlgebra 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [9a3f8284] + Random 
  [9e88b42a] + Serialization 
  [6462fe0b] + Sockets 
  [8dfed614] + Test 
  [4ec0a83e] + Unicode 

julia> using TypedTables
[ Info: Precompiling TypedTables [9d95f2ec-7b3d-5a63-8d20-e2491e220bb9]
```

Precompilation causes constant CPU load, but never finishes.

Tested on Ubuntu 16.04.

**Comment by giordano** [comment]
I think this is fixed by #47.  Perhaps a new stable release would help?  The last one is from November

**Comment by oschulz** [comment]
See #50
